R/lag_valideringsdatasett.R

```diff
@@ -74,39 +74,47 @@ lag_valideringsdatasett = function(d_reg, indvars) {
   # Lag oversikt over variabeltypar
   vartypar = map(d_reg[datavars], ~ class(.x)) %>%
     unname()
-  unike_vartypar = unique(vartypar)
 
   # Slå sammen elementer fra samme klass i en string
-  vartypar_sammenlagt = c()
-  for( i in seq_len(length(vartypar))){
-    vartypar_sammenlagt[i] = paste(vartypar[[i]], collapse = "_")
-  }
+  vartypar_sammenlagt = map_chr(vartypar, \(x) paste(x, collapse = "_"))
+  
+
+  d_vartyper = tibble::tibble(vartypar = vartypar, 
+                              vartypar_sammenlagt = vartypar_sammenlagt
+                              )
```

Comment:
Dataramma bør vel lagast direkte, og ikkje ved å fyrst laga to vektorar? Poenget var jo å sikra parallellitet, og at ikkje nokon seinare endrar rekkjefylgja i ein av vektorane, utan å veta at koden nedanfor er avhenging av henne.